src/content/episodes/11.md

```diff
@@ -0,0 +1,71 @@
+---
+title: 'Eleventy'
+emoji: 🦝
+recorded: 2021-11-03
+date: 2021-11-10
+duration: '59:35'
+description: |
+  Eleventy on static site generator (SSG),
+  eli se ottaa vastaan Markdown-tiedostoja
+  ja oksentaa ulos staattisia HTML-tiedostoja.
+  Mitä hyvää ja huonoa siinä on?
+---
+
+## Kuulumisia 👂
+
+- 0:35 Neli- ja kolmipäiväisiä työviikkoja
+- 1:45 Hannes: sähköläskipyöräilä
```

Comment:
```suggestion
- 1:45 Hannes: sähköläskipyöräily
```

Reply:
Good catch 👍 it's now "sähköläskipyöräilyä"